corehq/apps/locations/models.py

```diff
@@ -114,6 +114,17 @@ class LocationType(models.Model):
     )  # levels below this location type that we start expanding from
     _expand_from_root = models.BooleanField(default=False, db_column='expand_from_root')
     expand_to = models.ForeignKey('self', null=True, related_name='+', on_delete=models.CASCADE)  # levels above this type that are synced
+    _include_without_expanding = models.ForeignKey(
+        'self',
+        null=True,
+        related_name='+',
+        db_column='include_without_expanding',
+        on_delete=models.CASCADE
```

Comment:
good question. Replicated it from above, but I think that might be wrong (although it isn't behaving as I expected this  would with `CASCADE`, where deleting this deletes the referenced model, so I'm a bit confused why I added that in the first place).